server/src/main/java/org/opensearch/telemetry/TelemetrySettings.java

```diff
@@ -39,4 +44,11 @@ public boolean isTracingEnabled() {
         return tracingEnabled;
     }
 
+    public Settings getSettings() {
```

Comment:
This is not needed pass both Settings and ClusterSettings in getTelemetry() call.

Reply:
Updated